src/HtmlElement/HtmlElement.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HtmlElement = ({ element, children, ...props }) =>
+  React.createElement(element, props, children);
+
+HtmlElement.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node)
+  ]),
+  element: PropTypes.string
```

Comment:
Should we be restrictive of valid HTML elements here? Like`PropTypes.oneOfType`?

Reply:
Good call, no we should not be